docs/getting_started.md

```diff
@@ -0,0 +1,206 @@
+1. Your test results will most likely be three test failures:
+
+  ```PowerShell
+  Starting discovery in 1 files.
+  Discovery found 3 tests in 40ms.
+  Running tests.
+  Get-AzContainerRegistry: Resource group 'rg-test' could not be found.
+  [-] Verify Container Registry Exists.Should contain a container registry with the given name 537ms (530ms|7ms)
+    Expected a value, but got $null or empty.
+    at $exists | Should -Not -BeNullOrEmpty, \benchpress\examples\ContainerRegistry.Tests.ps1:15
+    at <ScriptBlock>, \benchpress\examples\ContainerRegistry.Tests.ps1:15
+  Get-AzContainerRegistry: Resource group 'rg-test' could not be found.
+  [-] Verify Container Registry Exists.Should contain a container registry with the given name 213ms (213ms|0ms)
+    Expected $true, but got $false.
+    at $exists | Should -Be $true, \benchpress\examples\ContainerRegistry.Tests.ps1:29
+    at <ScriptBlock>, \benchpress\examples\ContainerRegistry.Tests.ps1:29
+  New-AzResourceGroupDeployment: 1:48:19 PM - Error: Code=ResourceGroupNotFound; Message=Resource group 'rg-test' could not be found.
+  New-AzResourceGroupDeployment: The deployment validation failed
+  [-] Spin up , Tear down Container Registry.Should deploy a bicep file. 6.01s (6.01s|0ms)
+    Expected 'Succeeded', but got $null.
+    at $deployment.ProvisioningState | Should -Be "Succeeded", \benchpress\examples\ContainerRegistry.Tests.ps1:47
+    at <ScriptBlock>, \benchpress\examples\ContainerRegistry.Tests.ps1:47
+  Tests completed in 4.74s
+  Tests Passed: 0, Failed: 3, Skipped: 0 NotRun: 0
+  ```
+
+## Walkthrough of Test File
+
+Let's walkthrough the `ContainerRegistry.Tests.ps1` file to understand how BenchPress is used to test our Infrastructure as Code (IaC) and why our tests are failing.
+
+```PowerShell
+BeforeAll {
+  Import-Module "../BenchPress/Helpers/Azure/BenchPress.Azure.psd1"
+}
+
+Describe 'Verify Container Registry Exists' {
+  it 'Should contain a container registry with the given name' {
+    #arrange
+    $rgName = "rg-test"
+    $acrName = "acrbenchpresstest1"
+
+    #act
+    $exists = Get-AzBPContainerRegistry -ResourceGroupName $rgName -Name $acrName
+
+    #assert
+    $exists | Should -Not -BeNullOrEmpty
+  }
+}
+
+Describe 'Verify Container Registry Exists' {
+  it 'Should contain a container registry with the given name' {
+    #arrange
+    $rgName = "rg-test"
+    $acrName = "acrbenchpresstest1"
+
+    #act
+    $exists = Get-AzBPContainerRegistryExist -ResourceGroupName $rgName -Name $acrName
+
+    #assert
+    $exists | Should -Be $true
+  }
+}
+
+Describe 'Spin up , Tear down Container Registry' {
+  it 'Should deploy a bicep file.' {
+    #arrange
+    $resourceGroupName = "rg-test"
+    $bicepPath = "./containerRegistry.bicep"
+    $params = @{
+      name           = "acrbenchpresstest2"
+      location       = "westus3"
+    }
+
+    #act
+    $deployment = Deploy-AzBPBicepFeature -BicepPath $bicepPath -Params $params -ResourceGroupName $resourceGroupName
```

Comment:
The deployment and setup would probably be better demonstrated in a BeforeAll block to be consistent with best practices for organizing Pester Tests.

Reply:
Ah, yeah, fair point. I was thinking more along the lines of this being *only* documentation. But, I forgot that the source of the documentation is the original files. I'm OK with leaving it the way it is so that it is consistent with the files.